glusterd2/commands/snapshot/snapshot-activate.go

```diff
@@ -41,16 +41,11 @@ func validateSnapActivate(c transaction.TxnCtx) error {
 		}
 		fallthrough
 	case false:
-		brickStatuses, err := volume.CheckBricksStatus(vol)
+		brickinfos, err = snapshot.GetOfflineBricks(vol)
 		if err != nil {
+			log.WithError(err).Error("failed to get Brick status")
```

Comment:
Fix the error message to indicate correct failure ("failed to get offline bricks").

Reply:
done